### PR TITLE
Java CRONS wrapper support

### PR DESCRIPTION
**Pull request by adinauer**
Docs for Java CRONS wrapper support including upsert (which hasn't been released yet).

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

**Review by davidenwang — APPROVED** [review]
changes look good, we will merge when the java wrapper changes are live right?

**Comment by adinauer** [comment]
Yeah, let's wait for the SDK changes to be released before merging.

**Comment by adinauer** [comment]
https://github.com/getsentry/sentry-java/releases/tag/6.34.0 has been released.

**Pull request merged**
